lib/services/android-plugin-build-service.ts

```diff
@@ -0,0 +1,365 @@
+import * as path from "path";
+import * as shell from "shelljs";
+
+const xml2js = require("xml2js");
```

Comment:
I suggest to add a separate service that is wrapper for `xml2js` package and to inject the new service in constructor. (It's not a merge stopper)

Reply:
IMO we can hold off on that because `xml2js` is used only in `AndroidPluginBuildService` and there's no publicly exposed API, that can be used when the CLI is required.